examples/live/run.sh

```diff
@@ -195,10 +195,10 @@ python -m mpi4py `which pycbc_live` \
 --src-class-eff-to-lum-distance 0.74899 \
 --src-class-lum-distance-to-delta -0.51557 -0.32195 \
 --run-snr-optimization \
+--snr-opt-method differential_evolution \
 --snr-opt-di-maxiter 50 \
 --snr-opt-di-popsize 100 \
```

Comment:
Should these not also be passed via `--snr-opt-extra-opts`?

Reply:
I can do, would make args_to_string a lot cleaner